fastlane_bot/helpers/txhelpers.py

```diff
@@ -1001,15 +1001,31 @@ async def _get_layer_one_gas_fee(self, signed_transaction) -> Decimal:
         returns: Decimal
             The total fee (in gas token) for the l1 gas fee
         """
-        ethereum_base_fee = await self.ConfigObj.GAS_ORACLE_CONTRACT.caller.basefee()
-        fixed_overhead = await self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeOverhead()
-        dynamic_overhead = await self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeScalar()
+        ethereum_base_fee = await self._get_layer_one_gas_price()
+        fixed_overhead = await self._get_layer_one_fee_overhead()
+        dynamic_overhead = await self._get_layer_one_fee_scalar()
```

Comment:
I believe that returning all values from a single (async) function would be more efficient (otherwise, you're essentially running these functions in a sequential manner, awaiting for the completion of each function-call before moving to the next).